main/src/ca/uwaterloo/flix/api/Bootstrap.scala

```diff
@@ -681,14 +683,12 @@ class Bootstrap(val projectPath: Path, apiKey: Option[String]) {
       case Some(manifest) => manifest.modules
     }
 
-    Validation.mapN(flix.check()) {
-      root =>
-        HtmlDocumentor.run(root, packageModules)(flix)
-    }.toHardResult match {
-      case Result.Ok(_) =>
-        Validation.success(())
-      case Result.Err(errors) =>
-        Validation.toHardFailure(BootstrapError.GeneralError(flix.mkMessages(errors)))
+    val (result, errors) = flix.check()
+    result.foreach(HtmlDocumentor.run(_, packageModules)(flix))
```

Comment:
We should not run the HtmlDocumenter if there are errors.

Reply:
Ok but it did that before :)

Reply:
Really? Thats crazy.